pallets/elections/src/lib.rs

```diff
@@ -245,69 +254,69 @@ pub mod pallet {
             Ok(())
         }
 
-        /// Sets kick-out config, it has an immediate effect
+        /// Sets ban config, it has an immediate effect
         #[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
-        pub fn set_kick_out_config(
+        pub fn set_ban_config(
             origin: OriginFor<T>,
             minimal_expected_performance: Option<u8>,
             underperformed_session_count_threshold: Option<u32>,
             clean_session_counter_delay: Option<u32>,
```

Comment:
Let's add the possibility to set `ban_period` to follow the existing convention. We need it for e2e tests anyway.

Reply:
Added and also new extrinsic for changing elections openness.